src/PKSim.Infrastructure/ProjectConverter/v13/Converter12To13.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PKSim.Core;
+using System.Xml.Linq;
+
+namespace PKSim.Infrastructure.ProjectConverter.v13
+{
+   public class Converter12To13 : IObjectConverter
+   {
+      // 13.0 is not compatible with 12.0, but you don't need an explicit conversion to move forward.
+      // To satisfy the next converter, the object must pass through v13.0 conversion
```

Comment:
I guess this will change soon. Some features planned for V13 will require project conversion for sure :)

Reply:
Yeah I just put that for now.